module-2/backend/week_two.md

```diff
@@ -26,11 +36,33 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+`Team.find(4).owner`
+
 5. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+
+Teacher has many Students, a Student belongs to a Teacher
+
+```table students
+|      id    |
+|     name   |
```

Comment:
a belongs to should have a relationship on that table. A student will have a teacher_id on their table, not the way you have it